modules/tensor_mechanics/doc/content/source/postprocessors/SidesetReaction.md

```diff
@@ -0,0 +1,19 @@
+# Cavity Pressure Postprocessor
+
+!syntax description /Postprocessors/SidesetReaction
+
+## Description
+
+`SidesetReaction` computes the force $F$ along vector $\vec{e}$ acting on a sideset $\partial S$:
+
+\begin{equation}
+  F = \int_{\partial S} \vec{n}^T ~ \underline{\sigma}  \vec{e}  dS,
+\end{equation}
+
+where $\underline{\sigma}$ is the stress tensor.
+
+!syntax parameters /Postprocessors/SidesetReaction
+
+!syntax inputs /Postprocessors/SidesetReaction
+
+!syntax children /Postprocessors/SidesetReaction
```

Comment:
Apparently the checks for this are not resulting in failures, but we'll get errors from moosedocs if we don't have an AD version of the markdown page. I don't think we have a great system for dealing with this. Maybe make a really short page that links to the other one?

Reply:
When I build documentation without  `ADSidesetReaction.md` the system automatically generates an entry for it. When I add the markdown file, it's simply ignored.

Reply:
Really?! That wasn't our experience on another model we recently set up like this.